scripts/yaml_characters

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require 'pathname'
+
+exec('bin/rails', 'runner', $0, *ARGV) unless defined?(Rails)
```

Comment:
Another way to load rails is to require `environment.rb` which I think will do what you're trying to do here. So from a "plain ruby" shebang you'll get the full rails env